component-photo-gallery/src/Organisms/ImageList.js

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from 'react';
+import ImageItem from '../Molecules/ImageItem'
 
 export default class ImageList extends Component {
     render() {
+        const allImages = this.props.images.map(image => {
```

Comment:
So to get this test to pass, you will need to import your data and use it here:
        import data from your data file
        const allImages = data.map(image => {
this will get your test to pass!